terms.html

```diff
@@ -99,9 +98,10 @@
   </dd>
   <dt><dfn data-lt="presentations">presentation</dfn></dt>
   <dd>
-A set of one or more <a>verifiable credentials</a> transmitted by a holder.
+A digital artifact transmitted from the holder to the verifier to prove to the verifier that he
+possesses the appropriate <a>verifiable credentials</a>.
```

Comment:
I realize that the old text used "transmitted" as well, but given that this is a data model spec and not a protocol one, I think we should avoid using that word as a core part of defining a presentation. I recommend we change this language to:

"A digital artifact constructed by a holder that can be used to prove to a verifier that the holder possesses the appropriate verifiable credentials."

Side note: I also avoided the gender-specific pronoun on purpose there -- which we should also do.

Reply:
Changed language